### PR TITLE
add \c to prompt defined variables for displaying current time

### PR DESCRIPTION
**Pull request by nktks**
Hello.

I would be happy if the current time could be displayed in the prompt so that I can share the time of the prompt.
How about this?

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -198,6 +198,7 @@ Variables:
 * `\i` : Cloud Spanner Instance ID
 * `\d` : Cloud Spanner Database ID
 * `\t` : In transaction
+* `\c` : Current time in RFC3339 format
```
Comment (nktks) [inline README.md:201]: I couldn't find good variable name,  `\t` (for time) is used for `In transaction`, `\d` (for date) is used for `Database ID`,
I choose `\c` (for current time).

**Comment by yfuruyama** [comment]
Thank you for filing a pull request. Could you share more details of the use case for displaying current time in the prompt?

We have several ways to display current time (or transaction timestamp).

1. Call `SELECT CURRENT_TIMESTAMP();`

```
spanner> SELECT CURRENT_TIMESTAMP();
+--------------------------------+
|                                |
+--------------------------------+
| 2023-05-16T06:30:25.434472382Z |
+--------------------------------+
```

2. Use `-v` option to dump transaction timestamp.

```
$ spanner -v ...

spanner> select * from t limit 1;
+----+------+
| id | name |
+----+------+
| 1  | baz  |
+----+------+
1 rows in set (3.49 msecs)
timestamp:            2023-05-16T15:29:26.09011+09:00
cpu time:             2.24 msecs
rows scanned:         1 rows
...
```

Do these ways can't cover your use case?

**Comment by nktks** [comment]
@yfuruyama san

Thanks for your quick response and confirming this.
I want to share each statement execution time to other member. 
My usecase is covered by `-v` ...
So I'll try it.

Thank you!

**Comment by yfuruyama** [comment]
@nktks Thank you for your quick response!

Please feel free to reopen this if you find `-v` doesn't match your use case well. Thanks!

**Pull request closed without merging**